plugins/services/src/js/components/forms/ContainerServiceFormSection.js

```diff
@@ -23,92 +23,100 @@ const {DOCKER, NONE, MESOS} = ContainerConstants.type;
 
 const containerSettings = {
   privileged: {
-    label: <span>Grant Runtime Privileges</span>,
+    label: 'Grant Runtime Privileges',
     helpText: 'By default, containers are “unprivileged” and cannot, for example, run a Docker daemon inside a Docker container.',
     dockerOnly: 'Grant runtime privileges is only supported in Docker Runtime.'
   },
   forcePullImage: {
-    label: <span>Force Pull Image On Launch</span>,
-    helpText: 'Force Docker to pull the image before launching each task.',
+    label: 'Force Pull Image On Launch',
+    helpText: 'Force Docker to pull the image before launching each instance.',
     dockerOnly: 'Force pull image on launch is only supported in Docker Runtime.'
   }
 };
 
-const getArtifactsLabel = () => {
-  return (
-    <FieldLabel>
-      {'Artifact URI '}
-      <Tooltip
-        content="Provided URIs are passed to Mesos fetcher module and resolved at runtime."
-        interactive={true}
-        maxWidth={300}
-        scrollContainer=".gm-scroll-view"
-        wrapText={true}>
-        <Icon color="grey" id="circle-question" size="mini" />
-      </Tooltip>
-    </FieldLabel>
-  );
-};
-
 class ContainerServiceFormSection extends Component {
-  getArtifactsInputs(data = []) {
-    const errors = this.props.errors.fetch || [];
+  getArtifactsLabel() {
+    const tooltipContent = (
+      <span>
+        {'If your service requires additional files and/or archives of files, enter their URIs to download and, if necessary, extract these resources. '}
+        <a
+          href="https://mesosphere.github.io/marathon/docs/application-basics.html"
```

Comment:
This link doesn't tell me anything about artifacts. Do we have this in the marathon docs yet?